Find a functionally equivalent code:android.graphics.Matrix.postConcat:COMMENT
Method Modifier: public      
Comment:/**
 * Postconcats the matrix with the specified matrix.
 * M' = other * M
 */

@@ -1,3 +1,4 @@
 {
-    return native_postConcat(native_instance, other.native_instance);
+    native_postConcat(native_instance, other.native_instance);
+    return true;
 }

